Scripts/script-ZipFile.yml

```diff
@@ -0,0 +1,132 @@
+commonfields:
+  id: ZipFile
+  version: -1
+name: ZipFile
+script: |
+  import zipfile
+  import os
+  from os.path import isfile
+  try: # in order to support compression of the file
+      import zlib
+      compression = zipfile.ZIP_DEFLATED
+  except:
+      compression = zipfile.ZIP_STORED
+  filePath = None
+  fileEntryID = ''
+  zipName = None
+  # preparation for later version which will support several inputs
+  # if demisto.args().has_key('fileName'):
+  #   entries = demisto.executeCommand('getEntries', {})
+  #   for entry in entries:
+  #       fn = demisto.get(entry, 'File')
+
+  #       is_text = type(fn) in [unicode, str]
+  #       is_correct_file = demisto.args().get('fileName', '').lower() == fn.lower()
+  #       if is_text:
+  #           if demisto.args().has_key('fileName') and is_correct_file:
+  #               fileEntryID = entry['ID']
+  #               break
+  #       if fileEntryID == '':
+  #           errorMessage = ''
+  #           if demisto.args().has_key('fileName'):
+  #               demisto.results({
+  #                   'Type': entryTypes['error'],
+  #                   'ContentsFormat': formats['text'],
+  #                   'Contents': demisto.args().has_key('fileName') + ' not such file in war room'
+  #               })
+  #           sys.exit(0)
+
+  if demisto.args().has_key('entryID'):
+    fileEntryID = demisto.args().get('entryID')
```

Comment:
entryID is required field, so there is no need to check `has_key` you can just assume that there is such key